defaults/main.yml

```diff
@@ -8,6 +8,7 @@ roundcube_support_url: https://www.systemli.org/kontakt.html
 roundcube_user: roundcube
 roundcube_group: roundcube
 roundcube_user_shell: /bin/false
+roundcube_user_home_directory: "/var/www/{{ roundcube_domain }}"
```

Comment:
I think this approach is a bit confusing for people who don't face the issues you face. Maybe instead we could introduce a `roundcube_home` variable that defaults to `"{{ roundcube_user_home }}` and use that variable everywhere except for the task that creates the user? What do you think @t2d?